app-config-electron/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { WebPreferences } from 'electron';
+import { config } from '@app-config/main';
+import * as path from 'path';
+
+export function addAppConfigPreload(baseWebPreferences?: WebPreferences) {
```

Comment:
I take it we have to assume app config is loaded by this point?

What about passing the config object as an argument to this function?

Reply:
Yeah we do

I think that's better. What's the type of the config `ExportedConfig`?

Reply:
Yep, it should be. Naturally, this function can use a default value for the argument but making it explicit would be better imo.